module/modmanager/manager.go

```diff
@@ -633,11 +633,14 @@ func (mgr *Manager) ResolveImplicitDependenciesInConfig(ctx context.Context, con
 			// continue if this component is not being provided by a module.
 			continue
 		}
-		if !slices.ContainsFunc(conf.Modified.Modules, func(elem config.Module) bool {
+
+		//nolint:gocritic
+		deltaModules := append(conf.Modified.Modules, conf.Added.Modules...)
```

Comment:
unless we have access to https://pkg.go.dev/slices#Concat this is the safer way to do this

```suggestion
                lenModified, lenAdded := len(conf.Modified.Modules), len(conf.Added.Modules)
                deltaModules = make([]config.Module, lenModified, lenModified + lenAdded)
                copy(deltaModules, conf.Modified.Modules)
		deltaModules := append(deltaModules, conf.Added.Modules...)
```

Reply:
ah yeah I forgot slices work a bit funky (thanks for the article) this is definitely much safer! change made